template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/raz/RangerRazBaseConfigProvider.java

```diff
@@ -31,6 +37,15 @@ public String getServiceType() {
 
     @Override
     protected List<ApiClusterTemplateConfig> getRoleConfigs(String roleType, TemplatePreparationObject source) {
+        if (source.getProductDetailsView() != null && isRazTokenConfigurationSupported(source.getProductDetailsView().getCm())) {
+            if (CloudPlatform.AZURE == source.getCloudPlatform()) {
+                String safetyValveValue = getSafetyValveProperty("ranger.raz.bootstrap.servicetypes", "adls");
```

Comment:
Not a big deal but I suggest extracting `"ranger.raz.bootstrap.servicetypes"` as a constant.